L10-byteCodes/src/main/java/ru/otus/aop/proxy/LoggableFactory.java

```diff
@@ -0,0 +1,15 @@
+package ru.otus.aop.proxy;
+
+import lombok.SneakyThrows;
+
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Proxy;
+
+public class LoggableFactory<T extends Loggable> {
```

Comment:
Давайте просто уберем ` extends Loggable`, чтобы можно было получить любой прокси.

```suggestion
public class LoggableFactory<T> {
```